.github/workflows/release.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
```

Comment:
Have you looked into simplifying this workflow by creating a generic workflow that builds a given target and then calling the generic workflow with a matrix of targets? Are there some subtleties between targets that prevent doing so?

Reply:
I tried this but GitHub action does not give much reusability unless we go with a full bash script that we can invoke. You will not lot of `if` at the en of each pipeline. I tried factoring this into reusable action without much success. composite actions don't get all the context of the top-level action.

Reply:
Cool, just wanted to know if we tried and what the outcome was for developing future workflows. Thanks.